GitUI/CommandsDialogs/FormAbout.cs

```diff
@@ -21,6 +22,11 @@ public FormAbout()
 
             environmentInfo.SetCopyButtonTooltip(_copyTooltip.Text);
 
+            Color clrLink = ColorHelper.IsLightTheme() ? Color.Blue : Color.SkyBlue;
```

Comment:
While the links are hard to read in dark themes, this is a general problem and this increases inconsistency.
Some links use SystemColors.HotTrack instead (see for instance CommitInfo. Looks a little worse.
Even if I commented on this earlier, I propose to revert this change, at least for now.

Reply:
Why not use HotTrack in About too then? It is skyblue on my theme and looks good.
I changed to blue it in next commits (they aren't that visible now).

Reply:
We could change all to HotTrack, but there are quite some LinkLabel uses.
It is maybe better to use a custom control to set also visited links.
The color should probably be unset, to use the defaults.